regex-assembly/include/unix-shell-4andup.ra

```diff
@@ -83,7 +83,15 @@ adduser
 agetty
 alias@
 alpine@
+ansible
 ansible-playbook
+ansible-config
```

Comment:
Why would we list all the individual ansible commands, when we have naked `ansible` in the list?

Reply:
Agreed on this one. With `ansible` things should work already.

Maybe you want to also add `chef-client`, etc.

Reply:
I assumed you may have wanted all of the ansible commands in the list. But yes I agree, the regex should be a bit faster and simpler now. I'll add some chef/puppet commands in a sec.